### PR TITLE
LICENSE: use standard Apache V2.0 text

### PR DESCRIPTION
**Pull request by bmeverett**
closes hoodiehq/camp#37

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,201 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright {yyyy} {name of copyright owner}
```
Comment (gr2m) [inline LICENSE]: this should be `Copyright 2012-2016 the Hoodie Community and other contributors.`, see https://github.com/hoodiehq/hoodie/blob/master/LICENSE#L189

**Comment by bmeverett** [comment]
no problem. I have added a new commit

File: LICENSE (hunk 1 above)
Comment (bmeverett) [inline LICENSE]: sorry about that. I updated again.

Comment (gr2m) [inline LICENSE]: no problem at all :) Thanks for the fix

**Pull request merged**